#4: Added connection status indicator + icon

JustinWozasek opened the pull request:
+added icons and materials for "connected" and "not connected"
+added a display for icons at the right side of the button bar
+script "ConnectionStatusCheck" swaps icons depending if OSC is "Connected" or "Disconnected" using OscRemoteStatus

JustinWozasek committed:
Merge pull request #1 from Jindorf/master

Getting my Fork up to date with Master

JustinWozasek committed:
Merge branch 'master' of https://github.com/CptBlackFox/HoloOSCV2

JustinWozasek committed:
Added icons for checking connection status

There are now icons for connected (green signal bar) and not connected (red cross) at the right side of the buttons. When OSC is not connected, the cross is diplayed whereas after connecting the signal bar is displayed.

Jindorf commented:
Bug: Connection info isn't updated when connection is lost. 
Steps to reproduce:
1. open Reaper
2. open Connection in Reaper on port 8000
3. start the application in Unity
4. Press the UpdateReciever-Button, so the Reciever is automatically set to localhost:8000
 - the info-icon now updates to green
5. close the port in reaper (to simulate a lost connection)

Expected behaviour: The info-icon now updates to red
Actual behaviour: The info-icon stays green

Jindorf commented:
Possible improvements:

Checking for the connection in the Update-function is easy, but bad practice (performance suffers).
It would be better to get a notification when the connection succeeds / breaks.

JustinWozasek committed:
Merge remote-tracking branch 'upstream/master'

JustinWozasek committed:
#4: Requesting the status of connection with reaper now works

+The "mute" on channel 64 is now used to aks if reaper can receive and send OSC-messages.
-TODO: Display of corresponding icon to status not working properly, still looking for ways to fix the issue.

JustinWozasek committed:
#4: Fixed Bug regarding the display of icons

Icons are now displayed correctly on connectionstatus. Also the "Disconnected" Icon doesn't blink anymore at restart of the cycle.

JustinWozasek committed:
#4: Merging wip branch into master

#4: Fixed Bug "Connection info isn't updated when connection is lost." (Jindorf, 09.11.2019)

JustinWozasek committed:
Merge remote-tracking branch 'upstream/master'

Jindorf commented:
Material for Connection-status doesn't change even though a connection is present. 
![image](https://user-images.githubusercontent.com/21169298/70903443-31274480-1fff-11ea-9363-87550dc8e0ee.png)

JustinWozasek committed:
Merge remote-tracking branch 'upstream/master'

JustinWozasek committed:
#4: Fixed bug causing status not to update when appplication is started with muted Ch64 in Multiencoder

Jindorf committed:
Merge branch 'master' into master
